packages/react-cosmos-redux-proxy/src/index.jsx

```diff
@@ -1,16 +1,25 @@
 import React from 'react';
-import _ from 'lodash';
+import omit from 'object.omit';
```

Comment:
Is this new? I remember you had to use paths (e.g. `lodash/omit`) to benefit from this in the past.